config/sync/core.entity_view_display.node.stanford_opportunity.full.yml

```diff
@@ -0,0 +1,346 @@
+uuid: 6839c55e-a372-4242-8b54-441d4472c8cd
```

Comment:
We always use the "Default", not "full content". You can probably just copy the `contents` portion of this file into the `config/sync/core.entity_view_display.node.stanford_opportunity.default.yml` file